frequency.rb

```diff
@@ -0,0 +1,21 @@
+str = "mosheen"
+fre = Hash.new
+fre.default = 0
```

Comment:
@mohseengs what is the use of default here, we are using this property any where

Reply:
sir i have commented some lines, that are alternate ways of doing it
and there default will be used to make default value of any new key entered in hash.